plugins/woocommerce/package.json

```diff
@@ -577,9 +585,14 @@
 						"--shard=10/10"
 					],
 					"changes": [
-						"src/Blocks/**",
-						"templates/**",
-						"patterns/**"
+						"patterns/**/*.php",
+						"src/Blocks/**/*.php",
+						"templates/**/*.php",
+						"templates/**/*.html"
+					],
+					"onlyForDependencies": [
+						"@woocommerce/block-library",
+						"@woocommerce/e2e-utils"
```

Comment:
"@woocommerce/e2e-utils" is no longer used for long time now, interesting it's a dependency in blocks project. I'm sure it's not used though and can be removed.

Reply:
see #40117 and #40118

Reply:
Blocks E2Es are heavily using it, I double-checked and had similar thoughts.

Reply:
But regarding the deprecation path, let's drop this dependency from this config.

Reply:
Here we go: 280251c800ed999db9c4b76e7ef6c8f768b199d2

Reply:
> Blocks E2Es are heavily using it, I double-checked as had similar thoughts.

TIL! that package is old and using Puppeteer, blocks should not be using it.

Reply:
Actually, blocks are not using that package after all, `@woocommerce/e2e-utils` is mapped to `tests/e2e/utils`: https://github.com/woocommerce/woocommerce/blob/f4d80351eda1dab106db285827cf99ae5432103f/plugins/woocommerce/client/blocks/tsconfig.base.json#L68